src/App.js

```diff
@@ -25,7 +25,10 @@ class App {
 
 		this.model = wwtController.getModel();
 
-		this.revisionPopup = new RevisionPopupWidget();
+		const $overlay = $( '<div>' ).addClass( 'wwt-overlay' );
```

Comment:
If the welcome tour is active, we're already creating an overlay and attaching it to the body (see `browserextension.js` line 36). 

I'm wondering if it won't be easier to consolidate those two; it can even be in the form of creating the overlay as we set things up, and then pass the same overlay div to both of those; that way if the welcome popup appears we don't have two overlays attached to the DOM.

Reply:
I consolidated the two. An overlay was added to `Controller.js`so that only one overlay div is passed around and attached to the DOM.